readthedocs/projects/forms.py

```diff
@@ -357,6 +358,26 @@ def clean_conf_py_file(self):
             )  # yapf: disable
         return filename
 
+    def clean_build_config_file(self):
+        """
+        Validate user input to help user.
+
+        We also validate this path before using it to read a file, so this is
+        only considered helpful to a user.
```

Comment:
```suggestion
        Validate this path before using it to read a file.
```

No real idea what the original docstring here was trying to say about helping the user 2x?

Reply:
It was saying that the purpose of the validation is to help the user [not to do security]. I'll make it clearer.